docs/advanced/advanced_compatibility_reqs.md

```diff
@@ -0,0 +1,7 @@
+# Compatibility and Requirements
+
+For iSCSI single path users (RHEL only) verify that multipathing is installed and running.
```

Comment:
@roysahar could we let the storage documentation cover the multipath conf part, as we did until now?

Reply:
let's continue this in https://github.com/IBM/ibm-block-csi-driver/pull/453